apps/protocol-frontend/src/components/AttestationsTable.tsx

```diff
@@ -157,25 +160,48 @@ const AttestationsTable = ({
           if (status === 'Verified' || frameworkSettingThreshold === 0) {
             statusMapHover = 'Verified';
           }
+
+          if (
+            frameworkSettingThreshold === 0 &&
+            attestationThreshold === null
+          ) {
+            statusMapHover = 'Verified';
+          }
+
           if (status === 'Unverified') {
             statusMapHover = 'Unverified';
           }
 
           let pillStatusMap!: 'checkmark' | 'secondaryInfo' | 'primaryInfo';
+
           if (status === 'Verified' || frameworkSettingThreshold === 0) {
             pillStatusMap = 'checkmark';
           }
+
+          if (
+            attestationThreshold === null &&
```

Comment:
nit: consistent ordering with the conditional on 164

Reply:
Thanks! good call!